charts/redpanda/templates/servicemonitor.yaml

```diff
@@ -38,10 +38,22 @@ spec:
     scheme: https
     tlsConfig:
       insecureSkipVerify: true
```

Comment:
Should we skip this if we are applying the certs?

Reply:
It makes HTTPS request to IP address (without hostname) and check SANs for the certificate (and it does not match)